src/blocktree_processor.rs

```diff
@@ -53,22 +53,27 @@ fn ignore_program_errors(results: Vec<Result<()>>) -> Vec<Result<()>> {
         .collect()
 }
 
-fn par_execute_entries(bank: &Bank, entries: &[(&Entry, Vec<Result<()>>)]) -> Result<()> {
+fn par_execute_entries(bank: &Bank, entries: &[(&Entry, Vec<Result<()>>)]) -> (Result<()>, u64) {
     inc_new_counter_info!("bank-par_execute_entries-count", entries.len());
-    let results: Vec<Result<()>> = entries
+    let results_fees: Vec<(Result<()>, u64)> = entries
         .into_par_iter()
         .map(|(e, lock_results)| {
-            let old_results = bank.load_execute_and_commit_transactions(
+            let (old_results, fee) = bank.load_execute_and_commit_transactions(
                 &e.transactions,
                 lock_results.to_vec(),
                 MAX_ENTRY_IDS,
             );
             let results = ignore_program_errors(old_results);
             bank.unlock_accounts(&e.transactions, &results);
-            first_err(&results)
+            (first_err(&results), fee)
         })
         .collect();
-    first_err(&results)
+
+    let fee = results_fees.iter().map(|(_, fee)| fee).sum();
+    let mut results = Vec::new();
+    results.reserve_exact(results_fees.len());
```

Comment:
You shouldn't need this or the mut. map() or collect() should know to do this

Reply:
👍